types.go

```diff
@@ -139,6 +139,16 @@ type IndexInfo struct {
 
 	// Name of the similarity function used in indexing and queries.
 	SimilarityFunction string `json:"similarityFunction"`
+
+	NamespaceInfo map[string]NamespaceInfo `json:"namespaces"`
```

Comment:
I called this field `namespaces` in Python. Not sure what it will be called in vector-js, but it might be good to align all of them.